v2.0/paths/task_assignments__get_task_assignments_id.yml

```diff
@@ -22,3 +22,11 @@ responses:
       application/json:
         schema:
           $ref: '#/components/schemas/TaskAssignment'
+  404:
+    description: |-
+      Returns an error if the task assignment ID is incorrect or
```

Comment:
Do you think we might be better off rephrasing “incorrect” to something more descriptive? Maybe “if the task assignment for the given ID does not exist”?

Reply:
I don't feel super strongly about it. Happy to rephrase!

Reply:
I think “incorrect” is just a bit vague as is. If you don’t mind, let’s change it to the more specific.